.github/actions/ci/action.yml

```diff
@@ -27,7 +26,18 @@ runs:
       id: build
       run: dotnet build
 
+    - name: Conditional Build Frameworks For Tests
+      shell: bash
+      run: |
+        if [ "$RUNNER_OS" == "Windows" ]; then
+             echo "BUILDFRAMEWORKS=net471;net6.0;net8.0" >> "$GITHUB_ENV"
+        else
+             echo "BUILDFRAMEWORKS=net6.0;net8.0" >> "$GITHUB_ENV"
```

Comment:
The ubuntu runner no longer includes mono by default, and we don't really need to run those tests on linux/mac.